### PR TITLE
VIS removed season label 

### PR DESCRIPTION
**Pull request by augusta100**
# Summary

Removed season end label from resources graph. Added to graph description.

Screenshot 2021-01-14 at 20 28 38

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (lhl2617) [inline yarn.lock:1]: Delete

File: node_modules/.yarn-integrity [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "darwin-x64-83",
```
Comment (lhl2617) [inline node_modules/.yarn-integrity:2]: Delete

**Review by lhl2617 — APPROVED** [review]
Lgtm

File: website/src/components/Visualisations/Resources/Resources.tsx [hunk 3]
```diff
@@ -9,7 +9,8 @@ const Resources = (props: { output: OutputJSONType }) => {
       <p className={styles.text}>Resources over Time</p>
       <p>
         Select teams to show/hide by clicking the team at the top of the chart.
-        Use the slider at the bottom to change which turns are displayed.
+        Use the slider at the bottom to change which turns are displayed. The
+        dark dotted lines represent the end of a season.
```
Comment (lhl2617) [inline website/src/components/Visualisations/Resources/Resources.tsx:13]: Vertical?

File: yarn.lock (hunk 1 above)
Comment (augusta100) [reply yarn.lock:1]: Delete the line or the whole file?

**Pull request merged**